dashboard/config/initializers/delayed_job_config.rb

```diff
@@ -0,0 +1,3 @@
+# Prevents auto-deletion of failed jobs.
+# The failed jobs will be marked with non-null `failed_at`.
+Delayed::Worker.destroy_failed_jobs = false
```

Comment:
@davidsbailey This will cause failed EvaluateRubricJobs to pile up in the database - which is handy for troubleshooting, or if something downstream breaks and we want to retry a batch.

But we might want to follow this up with some code to scan the jobs table and clean up any failed jobs that have since had their projects evaluated. I'll start a Slack discussion about how the job failure & retry logic should work here.